optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ExamplesOverview/ExamplesOverview-section.adoc

```diff
@@ -118,9 +118,9 @@ and also in git under [path]_optaplanner/optaplanner-examples_.
 |<<vehicleRouting,Vehicle routing>> with time windows
 |* All of Vehicle routing
 * 1 shadow variable
-|* Entity <= `1000`
-* Value <= 1250
-* Search space <= `10^3000`
+|* Entity <= `2740`
+* Value <= `2795`
+* Search space <= `10^8380`
```

Comment:
According to my observation, this search space size (10^8380) is a result of having 2740 customers and 55 vehicles. What I can see here looks more like that `entities = customers` and `values = customers + vehicles`? Since vehicle is both an entity and a value and customers are values only, I'd expect it to be `entities = vehicles` and `values = vehicles + customers`. So
```
Entity <= 55
Value <= 2795
```
Wdyt?

Reply:
As it stands today:
- Customer is a genuine PlanningEntity
- Vehicle is a shadow PlanningEntity, so I don't count it towards the entities as those don't affect scalability

That being said, the new VRP model alternative with lists will impact this section somehow.